mobile/native/src/api_lndlc/mod.rs

```diff
@@ -0,0 +1,33 @@
+use anyhow::Result;
+use state::Storage;
+use tokio::runtime::Runtime;
+
+const ELECTRS_ORIGIN: &str = "tcp://localhost:50000";
+
+pub mod lndlc;
```

Comment:
:thinking: I don't get why there is an `lndlc` sub-module to the `api_lndlc` module. The separation doesn't seem to mean anything in this case.